core/platform/image/gae_image_services.py

```diff
@@ -67,7 +65,10 @@ def compress_image(image_content, scaling_factor):
         str. Returns the content of the compressed image.
     """
     if not constants.DEV_MODE:
-        height, width = get_image_dimensions(image_content)
+        img = Image.open(io.BytesIO(image_content))
```

Comment:
img --> image, throughout. Try to avoid abbreviations.

Reply:
Done